ui/components/app/modals/account-details-modal/account-details-modal.component.js

```diff
@@ -127,7 +138,16 @@ export default class AccountDetailsModal extends Component {
               showExportPrivateKeyModal();
             }}
           >
-            {this.context.t('exportPrivateKey')}
+            {process.env.MULTICHAIN ? (
+              <Text
+                variant={TextVariant.bodyMd}
+                color={TextColor.primaryDefault}
+              >
+                {this.context.t('showPrivateKey')}
+              </Text>
```

Comment:
Curious as to why this needs to be wrapped in a Text component? Should get away with just the string logic once the component-library Button is used?